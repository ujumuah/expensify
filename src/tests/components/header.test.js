import Navlink from '../../components/navlink'
import React from 'react'
import {shallow} from 'enzyme'

test('it should render navlink correctly',()=>{
const render  =  shallow(<Navlink/>)
 expect(render).toMatchSnapshot()
})