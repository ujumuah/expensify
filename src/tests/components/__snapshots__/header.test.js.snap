// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render navlink correctly 1`] = `
<div>
  <h1>
    PortFolio
  </h1>
  <ul>
    <li>
      <NavLink
        activeClassName="isActive"
        exact={true}
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="isActive"
        to="/create"
      >
        Create
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="isActive"
        to="/works"
      >
        Works
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="isActive"
        to="contact"
      >
        Contact
      </NavLink>
    </li>
  </ul>
</div>
`;
